wercker.yml

```diff
@@ -193,6 +193,16 @@ build:
           echo "build is skipped ..."
         fi
 
+  - script:
```

Comment:
Please comment out this section.
It will resolve CI failure and it more compact code.
Additionally I look at items in shell files as execution commands agnostic to environment of execution, I run same command on local - they should work.
So comment in CI config will mean that we are not ready to run it PR validation for now, but we can do this on local.
In some future there will be no wercker.sh , there will run.sh that has bunch of ready to use "scripts" inside for validation. All  of them should be executed on local, with exception of very specific build items. Good example of such concept is pitest.sh , it is executed in circleci but can be executed at any CI.

@strkkk, @pbludov , please review this concept

Reply:
I inverted the commented section. I commented our the YML and left the SH uncommented. It can be run locally if needed.

Reply:
> In some future there will be no wercker.sh , there will run.sh that has bunch of ready to use "scripts" inside for validation. 

IMO, we should avoid a super script just like we should avoid a super class. Too many commands in 1 file will make it become huge and possibly more problematic to maintain. CIs may change but the job the script is doing usually won't. What about splitting them by a type? We could have 1 script that contains everything regression related, etc... This is similar to the pitest script. Maybe any misc jobs that don't fit a type could be put into a plain `run` script.

Reply:
>What about splitting them by a type? 

This is exactly what I meant, but looks like lost in writing